src/main/java/org/jenkinsci/plugins/durabletask/executors/OnceRetentionStrategy.java

```diff
@@ -49,12 +54,13 @@
 
     private transient boolean terminating;
     
-    private int idleMinutes;
+    protected int idleMinutes;
```

Comment:
> This class was not designed for subclassing.

and this is mistake, 

>you would internally call the OnceRetentionStrategy

Why should i implement and wrap this class? 
1) Provide usable from UI class, that

> follow standard data binding idioms including a public getter (and Descriptor.getDisplayName would be used).

2) Make class re-usable, for migration purposes (or any other) i should subclass it. 

CloudRetentionStrategy do also bad thing by hiding idleTimeout that you in the same way duplicating in your strategy and that after de-serialisation duplicates xml entries.

> You seem to trying some other approach but I do not understand what that is, and I cannot really evaluate the PR without that background. (There was no description in the PR stating the use case.)

Provide standard Describable/Descriptor strategy if you want other plugins to use this strategy, that's all.